udpa/core/v1/resource_locator.proto

```diff
@@ -40,10 +40,10 @@ message ResourceLocator {
   // URI scheme.
   Scheme scheme = 1 [(validate.rules).enum = {defined_only: true}];
 
-  // Opaque identifiers for the resource. These are effectively concatenated
-  // with ‘/’ to form the non-query param path as resource ID. This may end
+  // Opaque identifier for the resource. Any '/' will not be escaped during URI
+  // encoding and will form part of the URI path. This may end
   // with ‘*’ for glob collection references.
-  repeated string id = 2 [(validate.rules).repeated = {min_items: 1}];
+  string id = 2 [(validate.rules).string = {min_len: 1}];
```

Comment:
Is `min_len: 1` correct?  Could this be empty?  Or this string expected to have a leading `/`?

Reply:
Yeah, I think you are right, we do allow empty resource IDs, will fix.

Reply:
Should we explicitly say that the value is not expected to contain a leading `/`?

Reply:
I don't think so; the URL encoding scheme in the comment above already states this.